lib/web_ui/dev/test/felt_test.dart

```diff
@@ -0,0 +1,14 @@
+import 'package:test/test.dart';
+import 'dart:io';
+import 'package:path/path.dart' as p;
+
+void main() {
+  test('felt.dart --usage exits 0', () async {
+    var dartExecutable = Platform.executable;
+    var testPath = Platform.script.toFilePath();
+    var feltPath = p.join(p.dirname(p.dirname(testPath)), 'felt.dart');
+    var result = await Process.run(dartExecutable, [feltPath, '--help']);
```

Comment:
This lint is not yet enabled for web code, but Flutter style guide forbids `var`. We always specify the type, and make the variable `final`, if possible.